.github/workflows/benchmark.yml

```diff
@@ -121,3 +121,23 @@ jobs:
         cd tests
         ./generate-programs.py
         ./run-programs.py
+
+  benchmarks:
+    name: derive, sign, verify performance
+    runs-on: benchmark
+    strategy:
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+
+    - name: Set up rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+
+    - name: chia-keys benchmark
+      run: |
+        cargo bench --all
```

Comment:
there's no upper limit imposed by CI right now, perhaps there should be, to catch performance regressions